DetourTileCache/Include/DetourTileCache.h

```diff
@@ -38,6 +38,7 @@ enum ObstacleState
 enum ObstacleType
 {
 	DT_OBSTACLE_CYLINDER,
+	DT_OBSTACLE_AABB,
```

Comment:
Please rename this back and use a new name for your addition.
For example: `DT_OBSTACLE_ORIENTED_BOX`.
To make it more clear you can add some comments that `DT_OBSTACLE_BOX` is an AABB and `DT_OBSTACLE_ORIENTED_BOX` is an OBB.